src/test/java/decodex/modules/ModuleManagerTest.java

```diff
@@ -1,39 +1,38 @@
 package decodex.modules;
 
-import decodex.data.exception.UnknownModuleException;
-import decodex.modules.base64.Base64Decoder;
-import decodex.modules.base64.Base64Encoder;
-import decodex.modules.hex.HexDecoder;
-import decodex.modules.hex.HexEncoder;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
+
+import decodex.data.exception.UnknownModuleException;
+import decodex.modules.base64.Base64Encoder;
 import org.junit.jupiter.api.Test;
 
 public class ModuleManagerTest {
 
     @Test
-    public void selectModule_availableModuleName_success() {
+    public void selectModule_base64encode_success() {
+        String moduleName = "base64encode";
         ModuleManager moduleManager = new ModuleManager();
         try {
-            assertTrue(moduleManager.selectModule("base64encode") instanceof Base64Encoder);
-            assertTrue(moduleManager.selectModule("base64decode") instanceof Base64Decoder);
-            assertTrue(moduleManager.selectModule("hexencode") instanceof HexEncoder);
-            assertTrue(moduleManager.selectModule("hexdecode") instanceof HexDecoder);
+            BaseModule module = moduleManager.selectModule(moduleName);
+            assertTrue(module instanceof Base64Encoder);
```

Comment:
Perhaps remove the surrounding `try-catch` and add the `UnknownModuleException` to the method signature of this test instead ?